DEVELOPMENT.md

```diff
@@ -101,41 +101,41 @@
 
 ## Test Running
 
--   Run all unit tests.
+-   Run all unit and integration tests.
 
     ```sh
-    bazel test //...
+    bazel test ...
     ```
 
--   Run a single unit test like `JavaCodeGeneratorTest.java`
+-   Run all unit tests.
 
     ```sh
-    bazel run //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest
+    bazel test $(bazel query ... | grep ^//:unit_)
     ```
 
--   Run a single integration test for API like `Redis`, it generates Java source
-    code using the Java microgenerator and compares them with the goldens files
-    in `test/integration/goldens/redis`.
+-   Run a single unit test like `JavaCodeGeneratorTest.java`
 
     ```sh
-    bazel test //test/integration:redis
+    bazel run :unit_com_google_api_generator_engine_JavaCodeGeneratorTest
     ```
 
--   Run all unit and integration tests.
+-   Update unit test golden files, for example `JavaCodeGeneratorTest.java`:
```

Comment:
Is there just one such "golden" unit test? It seems odd. @vam-google WDYT?

Reply:
Yes, this is strange (the previus `bazel run` line is also odd). @chanseokoh can you please describe what is going on here?

Basically there must be multiple golden unit tests with their own names and `JavaCodeGeneratorTest` does not even look  like a golden test but rather a regular unit test.

Reply:
> Yes, this is strange (the previus `bazel run` line is also odd).

`bazel run` is a typo. Should be `bazel test`.

> Basically there must be multiple golden unit tests with their own names and `JavaCodeGeneratorTest` does not even look like a golden test but rather a regular unit test.

Not sure I follow this. Things are working basically the same, just with slightly different names.

- Running tests:
   - old: `bazel test //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest`
   - new: `bazel test :com/google_api_generator_engine_JavaCodeGeneratorTest`

- Updating goldens:
   - old: `bazel run //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest_update`
   - new: `bazel run :update_com/google_api_generator_engine_JavaCodeGeneratorTest`